vnext/Chakra/ChakraCoreDebugger.h

```diff
@@ -91,6 +91,13 @@ class DebugProtocolHandler
 
     return result;
   }
+
+  JsErrorCode GetConsoleObject(JsValueRef *consoleObject)
```

Comment:
Should this be `JSValueRef* consoleObject)`? I see `void* callbackState` here so we should follow the desired convention in the code.

Reply:
ChakraDebugProtocolHandler.h does take JSValueRef* as parameter.